.gitignore

```diff
@@ -1,3 +1,4 @@
+.idea
```

Comment:
This changes should be reverted. It's too personal for different users.

I suggest you to add `.idea` to the global `.gitignore` on your host machine if you always use PhpStorm for developing. Check https://help.github.com/articles/ignoring-files/#create-a-global-gitignore